pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -106,11 +106,15 @@ func (s *Service) createInstance(machine *actuators.MachineScope, bootstrapToken
 		Role:        aws.String(machine.Role()),
 	})
 
+	var err error
 	// Pick image from the machine configuration, or use a default one.
 	if machine.MachineConfig.AMI.ID != nil {
 		input.ImageID = *machine.MachineConfig.AMI.ID
 	} else {
-		input.ImageID = s.defaultAMILookup(machine.Region())
+		input.ImageID, err = s.defaultAMILookup("ubuntu", "18.04", machine.Machine.Spec.Versions.Kubelet)
```

Comment:
This looks fine as a first approach, but can we put a TODO here about ubuntu 18.04 being hardcoded?

Reply:
Actually, re-read this. this is fine.